src/Configuration.php

```diff
@@ -235,4 +239,20 @@ public function getBaseUrl()
     {
         return $this->messageFactory->getBaseUrl();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setSSLVerifyPeer($verify)
```

Comment:
Can you use camel case here and the same for the getter? I don't use upper case even if the word is an abbreviation.